mirrord-layer/src/error.rs

```diff
@@ -125,6 +128,8 @@ impl<'a, T> From<std::sync::PoisonError<std::sync::MutexGuard<'a, T>>> for Layer
     }
 }
 
+pub(crate) type Result<T> = std::result::Result<T, LayerError>;
```

Comment:
Is it good practice to call it Result? I'm assuming most people going over the code would automatically assume it's std::Result and not realize there's a LayerError hidden there.

Reply:
AFAIK It's more than common. https://doc.rust-lang.org/std/io/type.Result.html for example - many modules have their own `Result` aliases, and so do we.

Reply:
P.S They can know it's not `std::Result` by seeing that only one generic is specified, and not two.